packages/charting/src/components/LineChart/LineChart.types.ts

```diff
@@ -1,22 +1,9 @@
 import { ITheme, IStyle } from '../../Styling';
 import { IStyleFunctionOrObject } from '../../Utilities';
+import { IDataPoint } from '../../types/IDataPoint';
 
 export interface ILineChart {}
 
-export interface IDataPoint {
```

Comment:
All of these are technically breaking changes since they used to be part of the top level API surface

Reply:
for a v0 package. It's fine.